src/hermes/cli.py

```diff
@@ -157,3 +157,4 @@ def main(ctx: click.Context, *args, **kwargs) -> None:
 main.add_command(workflow.process)
 main.add_command(workflow.deposit)
 main.add_command(workflow.postprocess)
+main.add_command(workflow.clean)
```

Comment:
This way, clean is *always* run when `hermes` is called without an explicit sub-command. I'm not sure this is what we really want...

Reply:
My proposal:
- make `clean` the first command
- add a `click.option("--clean", is_flag=True, default=False)` to the main. This way, clean is automatically run as first command as if you call `hermes --clean`.

Reply:
To the first point:  What does the `invoke_without_command` parameter in the decorator mean?
https://github.com/hermes-hmc/workflow/pull/108/files#diff-9c665150c60db6c7a4564d46b1beb793fc2ec2bb2df11aeae0e132c0e8c5bf99R111
I thought setting it to `False` would make it not run, but this lead to some error. I the changed it back and forgot about it, sorry.

Reply:
To the second point: I think we do not want to run it automatically before other commands. We also do not want to run it only when other commands are run either, it should be available as a standalone action which one can run, e.g., after completing a publication process.

Reply:
Ok, I think I see not how `click` works. I implemented your suggestion.